src/Simplex/Chat/Terminal/Output.hs

```diff
@@ -132,6 +159,54 @@ runTerminalOutput ct cc@ChatController {outputQ, showLiveItems, logFilePath} = d
         _ -> pure ()
     logResponse path s = withFile path AppendMode $ \h -> mapM_ (hPutStrLn h . unStyle) s
 
+responseNotification :: ChatResponse -> Maybe (Maybe ActiveTo, Notification)
+responseNotification = \case
+  CRNewChatItem u (AChatItem _ _ (DirectChat ct) ChatItem {chatDir = CIDirectRcv, content = CIRcvMsgContent mc, formattedText}) ->
+    let c = viewContactName ct
+     in cNtf u ct (ActiveC c) (c <> "> ") (msgText mc formattedText)
+  CRNewChatItem u (AChatItem _ _ (GroupChat g) ChatItem {chatDir = CIGroupRcv m, content = CIRcvMsgContent mc, formattedText}) ->
+    gNtf u g (ActiveG $ viewGroupName g) (fromGroup_ g m) (msgText mc formattedText)
+  CRContactConnected u ct _ ->
+    let c = viewContactName ct
+     in cNtf u ct (ActiveC c) (c <> "> ") "connected"
+  CRContactAnotherClient u ct ->
+    let c = viewContactName ct
+     in toMaybe (contactNtf u ct) (Nothing, Notification {title = c <> "> ", text = "connected to another client"})
+  CRContactsDisconnected srv _ -> serverNtf srv "disconnected"
+  CRContactsSubscribed srv _ -> serverNtf srv "connected"
+  CRReceivedGroupInvitation u g ct _ _ ->
+    let c = viewContactName ct
+     in cNtf u ct (ActiveC c) ("#" <> viewGroupName g <> " " <> c <> "> ") "invited you to join the group"
+  CRUserJoinedGroup u g _ ->
+    let gn = viewGroupName g
+     in gNtf u g (ActiveG gn) ("#" <> gn) "you are connected to group"
+  CRJoinedGroupMember u g m ->
+    let gn = viewGroupName g
+     in gNtf u g (ActiveG gn) ("#" <> gn) ("member " <> viewMemberName m <> " is connected")
+  CRConnectedToGroupMember u g m _ ->
+    let gn = viewGroupName g
+     in gNtf u g (ActiveG gn) ("#" <> gn) ("member " <> viewMemberName m <> " is connected")
```

Comment:
host and invitee are ok, but this seems too extreme. It doesn't change comparing to current implementation though, unlike CRReceivedGroupInvitation
```suggestion
     in gNtf u g Nothing ("#" <> gn) ("member " <> viewMemberName m <> " is connected")
```